src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1057,7 +1058,7 @@ private IParameterInitializer CreateBoundParameterEqualsValueOperation(BoundPara
         private IBlockStatement CreateBoundBlockOperation(BoundBlock boundBlock)
         {
             Lazy<ImmutableArray<IOperation>> statements =
-                new Lazy<ImmutableArray<IOperation>>(() => boundBlock.Statements.Select(s => Create(s)).Where(s => s.Kind != OperationKind.None).ToImmutableArray());
```

Comment:
>Select(s => Create(s)).Where(s => s.Kind != OperationKind.None).ToImmutableArray()); [](start = 81, length = 84)

I do not feel comfortable about this change, it is likely to have unintended consequences. If the goal is to keep ```fixed``` statement in the list, we should do exactly that, I believe.